### ISSUE TITLE
State minimum required version of Ansible Lint

### ISSUE THREAD

**Issue by mforkel** [ansible/vscode-ansible#753]
### Problem

It seems the extension requires at least version 5.0 of ansible-lint. This should be stated in the documentation. Currently, Ansible 2.9+ and Ansible Lint are listed as requirements. 

### Solution

Add the minimum required version for Ansible Lint in the requirements section of README.md

**Comment by ssbarnea** [comment]
I agree, we should at least give a clear error when ansible-lint is older than the minimum version we require and disable its integration, likely a WARN even ERROR statusbar badge in that case would be appropiate.
